### DISCUSSION TITLE
Using without DACs

### DISCUSSION THREAD

**Discussion by ProHarshit** [pschatzmann/arduino-espeak-ng#13] [Q&A] [answered]
Hi
Can I use this project without external DACs and amplifiers? 
If yes, then please suggest how.

**Comment by pschatzmann** [comment, marked as answer]
Just use one of the many available [output classes](https://github.com/pschatzmann/arduino-audio-tools/wiki/Audio-Sources-and-Sinks):
e.g.
- AnalogAudioStream 
- PWMStream
- File
- etc

**Reply by ProHarshit** [reply]
Can you give an example code please?

**Reply by pschatzmann** [reply]
Just have a look at the AudioTools project: there are really a lot of examples

**Reply by ProHarshit** [reply]
Is this the one example? 
https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-stream/streams-generator-analog/streams-generator-analog.ino

**Comment by pschatzmann** [comment]
This is an example that shows how to output a signal to the internal dac, as indicated in the Readme for this example

**Reply by ProHarshit** [reply]
Can you suggest an example for this please? 
Also thanks for your valuable time.

**Reply by ProHarshit** [reply]
Ok sorry. I understood. I will try this class for sure
